### PR TITLE
Feature/scroll-buttons

### PR DESCRIPTION
**Pull request by kellydinneen**
#### Description
Adds buttons to poster container that control scroll behavior.

- uses React hook`useRef()` to reference movie container element
- uses `element.scrollLeft` (where `element` is replaced by `scroller.current` ref)
- _Resource on this specific functionality:_ https://medium.com/codezillas/interacting-with-dom-elements-in-react-e7e409d7874e
- _Documentation on Refs in general:_ https://reactjs.org/docs/refs-and-the-dom.html

Styles hover effects for posters.

- posters now grow and slightly brighten on hover

Tweaks styling for header and footer.

- adds accent color for header/footer border and site title 
- shrinks header height

#### Type of change

- [x] Feature 
- [ ] Bug fix 
- [ ] Housekeeping (e.g. README, file naming)
- [ ] Refactor: 

  - [ ] *naming*
  - [ ] *DRYing*
  - [ ] *SRPifying*
  - [ ] *Other:*

#### Functionality area

- [x] styling
- [x] React component
- [ ] testing

#### Context
We wanted to replicate the scrolling functionality in our design inspiration: movie streaming sites like Kanopy, Mubi, and Netflix. These sites have button controls for scrolling through film posters. 

#### Where a reviewer should start

 - Movies.js, line 9

#### How to test this change
- no testing suite built yet, but can try scrolling functionality by running site and clicking on left/right arrows next to poster container

#### Next Steps/Loose Ends

- make scrolling distance more responsive

### CODE REVIEW

File: src/components/Movies.js [hunk 1]
```diff
@@ -2,8 +2,55 @@ import React from 'react';
...
+//   }
+// }
+
 const Movies = ({movies}) => {
 
+  const scroller = React.useRef();
+
+  const scrollLeft = () => {
+    console.log(scroller.current);
+    scroller.current.scrollLeft -= 400;
```
Comment (kellydinneen) [inline src/components/Movies.js]: Can only specify scroll distance in PIXELS, which is annoying for responsiveness. I think we'll have to use some conditionals based on window size if we want the button-facilitated scroll to be responsive :/

File: src/components/Poster.css [hunk 2]
```diff
@@ -1,4 +1,11 @@
 .poster {
   height: 30vh;
   margin: 2vh;
+  filter: drop-shadow(5px 5px 10px #434343);
+  transition: all .2s ease-in-out;
```
Comment (kellydinneen) [inline src/components/Poster.css:5]: transition for hover effect so that the user feels smooooothe

File: src/components/Movies.js [hunk 3]
```diff
@@ -2,8 +2,55 @@ import React from 'react';
 import Poster from './Poster';
 import './Movies.css';
 
+// class Movies extends React.Component {
+//   constructor({movies}) {
+//     super({movies})
+//     this.movies = {movies};
+//     this.scroller = React.createRef();
```
Comment (kellydinneen) [inline src/components/Movies.js]: I was initially getting errors while using `ref` in a functional component, so I tried to make `Movies` a class...then I discovered better documentation on refs and the function works fine 🤘

Comment (kellydinneen) [reply src/components/Movies.js]: for reasons I still don't understand, we have to use `React.createRef()` inside class components and `React.useRef()` inside functional components. Also notable: we can never assign a `ref` TO a functional component, but we can assign a `ref` TO a class component 🤨

File: src/components/Movies.js [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+import scrollLeftButton from './scroll-left.png';
+import scrollRightButton from './scroll-right.png';
 import './Movies.css';
 
 const Movies = ({movies}) => {
 
+  const scroller = React.useRef();
+
+  const scrollLeft = () => {
+    console.log(scroller.current);
```
Comment (lbmerchant93) [inline src/components/Movies.js:12]: I sure we wont miss it later to take this out but just making a note that there was a console.log left in! (Actually glad you left it so I could see more about what you added!)

File: src/components/Movies.js [hunk 5]
```diff
@@ -1,9 +1,22 @@
...
+    scroller.current.scrollLeft -= 600;
+  }
+
+  const scrollRight = () => {
+    scroller.current.scrollLeft += 600;
```
Comment (lbmerchant93) [inline src/components/Movies.js:17]: Do these number still do the same amount even if the window is bigger? I think these are a measurement of pixels, so if the window was say 300px wide, would it skip basically a whole section?

File: src/components/Movies.js [hunk 6]
```diff
@@ -15,8 +28,12 @@ const Movies = ({movies}) => {
   })
 
   return (
-    <div className='movies-container'>
-      {moviePosters}
+    <div className='movies'>
+      <img className="scrollButton" onClick={scrollRight} src={scrollLeftButton} onClick={scrollLeft}/>
```
Comment (lbmerchant93) [inline src/components/Movies.js:32]: Love the buttons addition! feels more like streaming services I've have used

File: src/components/Movies.js [hunk 7]
```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import Poster from './Poster';
+import scrollLeftButton from './scroll-left.png';
+import scrollRightButton from './scroll-right.png';
 import './Movies.css';
 
 const Movies = ({movies}) => {
 
+  const scroller = React.useRef();
```
Comment (lbmerchant93) [inline src/components/Movies.js:9]: Great work on figuring out how to "useRef"

**Review by lbmerchant93 — APPROVED** [review]
Amazing job!! React is really making it come along so quickly!!

**Pull request merged**